### PR TITLE
[fix](mtmv)mtmv add case and fix bug

### PR DESCRIPTION
**Pull request by zddr**
- allow add roll up
- fix auto bucket bug
- Add more cases

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

File: regression-test/suites/mtmv_p0/test_create_mv_mtmv.groovy [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import org.junit.Assert;
+
+suite("test_create_mv_mtmv","mtmv") {
```
[the remark this answers is not in the document]
Comment (zddr) [reply regression-test/suites/mtmv_p0/test_create_mv_mtmv.groovy:20]: yes

File: regression-test/suites/mtmv_p0/test_light_schema_change_mtmv.groovy [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import org.junit.Assert;
+
+suite("test_light_schema_change_mtmv","mtmv") {
```
[the remark this answers is not in the document]
Comment (zddr) [reply regression-test/suites/mtmv_p0/test_light_schema_change_mtmv.groovy:20]: The materialized view does not allow schema changes. This only applies to light_schema_change and does not affect the normal functionality of the materialized view

**Review by zfr9527 — APPROVED** [review]

**Pull request merged**